citestwheel.Dockerfile

```
@@ -153,7 +153,7 @@ pyenv rehash
 EOF
 
 # Install latest gha-tools
-RUN wget https://github.com/rapidsai/gha-tools/releases/latest/download/tools.tar.gz -O - \
+RUN -q wget https://github.com/rapidsai/gha-tools/releases/latest/download/tools.tar.gz -O - \
```

```suggestion
RUN wget -q https://github.com/rapidsai/gha-tools/releases/latest/download/tools.tar.gz -O - \
```

Looks like this was a mistake.